samples/word/07-custom-properties/custom-properties-access.yaml

```diff
@@ -1,24 +1,26 @@
-id: word-basics-custom-properties-access
+id: word-basics-custom-properties-access-
 name: Get Custom Document Properties
 description: Shows how to retrieve the current document's custom properties.
 author: Microsoft
 host: WORD
 api_set:
     WordApi: 1.1
 script:
-    content: |4
-          $("#run").click(getProperties);
+    content: |
+        $("#run").click(getProperties);
 
-            function getProperties() { 
-                Word.run(function (context) {
-                    var customDocumentProperties = context.document.properties.customProperties;
-                    context.load(customDocumentProperties);
-                    return context.sync()
-                        .then(function () {
-                            console.log(customDocumentProperties.items.length);
-                         })
-                 })
-            }
+        function getProperties() {
+            Word.run(function (context) {
+                var customDocumentProperties = context.document.properties.customProperties;
+                context.load(customDocumentProperties);
+                return context.sync()
+                    .then(function () {
+                        console.log(customDocumentProperties.items.length);
```

Comment:
yeap i think i will just leave the other example, who actually shows how to create and retrieve custom doc props. this is basically a dupe...